tests/integration/apigateway/conftest.py

```diff
@@ -168,3 +168,9 @@ def _import_apigateway_function(*args, **kwargs):
 
     for rest_api_id in rest_api_ids:
         delete_rest_api(aws_client.apigateway, restApiId=rest_api_id)
+
+
+@pytest.fixture
+def apigw_add_transformers(snapshot):
+    snapshot.add_transformer(snapshot.transform.key_value("id"))
```

Comment:
just a note, beware when using a straight `key_value` transformer with `id`, as multiple different kind of resources share this key, which can have weird effect in the snapshots... I've been bitten multiple times and I tend to use `json_path` now for those, to not mix up. When snapshotting only one kind of resource, it does make perfect sense though! 👍

Reply:
Agreed, and sometimes it doesn't catches the eye especially on initial tests.

Reply:
Great catch, 👍  thanks for the pointer! Updated..